main.go

```diff
@@ -94,6 +97,13 @@ See documentation at: https://github.com/ipfs/bifrost-gateway/#readme`,
 			log.Fatalf("Unable to start. bifrost-gateway requires either PROXY_GATEWAY_URL or STRN_ORCHESTRATOR_URL to be set.\n\nRead docs at https://github.com/ipfs/bifrost-gateway/blob/main/docs/environment-variables.md\n\n")
 		}
 
+		// TODO: create trace provider with correct exporters and all information
+		// we may need.
+		tp := trace.NewTracerProvider()
```

Comment:
Kubo has a very intricate trace provider creation. Not really sure what we want to do here. This is just the default, I think.

Reply:
@hacdias I believe we want the same thing – enabling additional trace reporters as opt-in via env vars.
We want Jaeger so we can test with Jaeger UI, but users will benefit from having other options available.  

Perhaps copy tracing.go from Kubo. All the tracing options from https://github.com/ipfs/kubo/blob/master/docs/environment-variables.md#tracing should be added to https://github.com/ipfs/bifrost-gateway/blob/main/docs/environment-variables.md.